### DISCUSSION TITLE
https://goreleaser.com/includes/templates/ is empty

### DISCUSSION THREAD

**Discussion by stdedos** [goreleaser/goreleaser#5004] [Q&A] [answered]
Some "Templates" links lead to the empty https://goreleaser.com/includes/templates/ page, including search

![image](https://github.com/user-attachments/assets/0bf53db3-2fd6-449e-adcb-496aff0111f6)

**Comment by caarlos0** [comment]
ahh, good catch! thanks!

**Reply by caarlos0** [reply, marked as answer]
should be fixed by https://github.com/goreleaser/goreleaser/pull/5006
